doc/specs/wiki.md

```diff
@@ -0,0 +1,315 @@
+# Wiki Specs
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+The site contains a variety of users with a vast wealth of knowledge. The wiki is a place where users can put their knowledge to work, and write articles for users to use. These may be supplementary to other documents on the site, or standalone documents such as guides.
+
+## Business rules
+
+* Articles must be able to be read/write locked based on user class.
+* Users with the wiki administration permission must have full access to all articles.
+* Article creation must be associated with a permission.
+* Duplicate aliases, tags, and dedicated terms must be unique and not empty.
+
+## Use cases
```

Comment:
Need a use case for viewing revisions on an article, as well as rolling back to a revision. Suggestion to make a separate spec for revisions since it it generic behavior not limited to wikis. You could just mention that wikis have revisions, what permissions are needed to manage them, and any extra info like what to track.

Same thing goes for comments.